### DISCUSSION TITLE
Temperature and Humidity Sensor does not update data

### DISCUSSION THREAD

**Discussion by paulmouse** [tuya/tuya-home-assistant#177] [Q&A]
Temperature and Humidity Sensor (model MIR-TE200-WF) - wifi mode.

Works fine in Tuya app. After pairing, the data appeared in HomeAssistant, but since then froze on the same parameters.
I look at the log, every minute there is such a record:
`2021-07-20 07:05:49 DEBUG (Thread-10) [tuya iot] _on_log: Sending PINGREQ`
`2021-07-20 07:05:49 DEBUG (Thread-10) [tuya iot] _on_log: Received PINGRESP`
What could be the reason?
Thanks in advance!

**Comment by iotinkognito** [comment]
Raise as issue rather than discussion,
